edit_rule_ui.gs

```diff
@@ -0,0 +1,177 @@
+/**
+ * Creates edit rule UI and return the card
+ * @param {Object} e is the Event Object 
+ *    which contains information about the context
+ * @return {CardService.Card} The card to show to the user
+ */
+function editRule(e) {
+  var ruleNumber = parseInt(e.parameters.Number);
+  var files = DriveApp.getFilesByName('Admin Rules For Apps Scripts');
```

Comment:
Use constants which you created for file name.

Reply:
updated